ethstats/ethstats.go

```diff
@@ -485,6 +579,14 @@ func (s *Service) reportLatency(conn *websocket.Conn) error {
 	if err := s.sendStats(conn, actionNodePing, ping); err != nil {
 		return err
 	}
+	// Proxy needs a delegate send here to get ACK
+	if s.backend.ProxiedPeer() != nil {
```

Comment:
I wonder if for basic checks like this to determine if this is a proxy or a proxied validator, we can instead use a boolean that's exposed from backend. It would help make this more readable too